### ISSUE TITLE
Problem in manage participants 

### ISSUE THREAD

**Issue by nclaidiere** [open-cogsci/omm-server#129]
In the attached you see that not all participants are assigned to the experiment but the message says 'no more participants available'.

It seems like this issue relates to others concerning the assignment of participants to experiments and the management of priorities.

![problem with the way participants are managed](https://user-images.githubusercontent.com/50901388/171214103-73d268dd-338e-48b7-a6ee-5bb18afe3f22.PNG)

**Comment by dschreij** [comment]
Right, that indeed appears to be incorrect and needs to be investigated. Did this participant get added after the experiment was created?

**Comment by nclaidiere** [comment]
No, it was added with the others but then died so was removed. You can't see it on the picture but there are other unselected participants below.

**Comment by dschreij** [comment]
I tried everything I could locally of which I thought it would trigger this problem, but haven't been able to.
If possible, please provide me some exact steps to take with which I can reproduce this issue (maybe also with a screen recording which you can make with Windows key + G on a Win 10 or 11 PC). I'm treading in the dark here of what could cause this issue.

**Comment by nclaidiere** [comment]
I have no idea on how to trigger it, the best would have been to look on the server when the error was present but I think sebastiaan wiped that out?
